src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -397,23 +848,10 @@ public static LdaTransform Create(IHostEnvironment env, ModelLoadContext ctx, ID
                     // <remainder handled in ctors>
                     int cbFloat = ctx.Reader.ReadInt32();
```

Comment:
>cbFloat [](start = 24, length = 7)

I think we might not need this any longer. We should ask if it's worth it to remove from the protocol.

Reply:
for now I am keeping this as is . prioritizing other comments, esp the one related to exposing LdaState info

---
In reply to: [231649602](https://github.com/dotnet/machinelearning/pull/1410#discussion_r231649602) [](ancestors = 231649602)